Lib/test/test_import/__init__.py

```diff
@@ -91,12 +91,12 @@ def test_from_import_missing_attr_has_name_and_path(self):
         self.assertRegex(str(cm.exception), r"cannot import name 'i_dont_exist' from 'os' \(.*os.py\)")
 
     def test_from_import_missing_attr_has_name_and_so_path(self):
-        import select
+        import _hashlib
```

Comment:
It's pretty easy for `_hashlib` to just not exist, for instance if OpenSSL is not available.  How about `_testcapi`?

Reply:
Okay, after talking with @doko42 we think `_testcapi` is a good choice.  We need to find one that will always be built *and* is not a built-in (on Debian).

Reply:
Same in Fedora; `_testcapi` sounds like a good choice.
(It's always built, as a shared library. Then it's packaged in `python3-tests` so it's not necessarily installed on all systems.)